pombola/core/views.py

```diff
@@ -128,6 +144,12 @@ def get_context_data(self, **kwargs):
         context = super(BasePlaceDetailView, self).get_context_data(**kwargs)
         context['place_type_count'] = models.Place.objects.filter(kind=self.object.kind).count()
         context['related_people'] = self.object.related_people()
+        if settings.ENABLED_FEATURES['projects']:
+            # The number of projects associated with the place is used
+            # in the link text in the object_menu_links:
+            context['projects_sub_link_text'] = 'CDF Projects ({0})'.format(
```

Comment:
That's reminded me - I've just put i18n of Pombola on the agenda for discussion next week. I'll change it to `'...{number}...'.format(number=...)`

Hooray for `blocktrans asvar` then! It does seem silly that this ends up in the view.